libraries/networking/src/udt/Connection.h

```diff
@@ -73,10 +73,12 @@ class Connection : public QObject {
 
 signals:
     void packetSent();
+    void connectionInnactive(HifiSockAddr sockAdrr);
```

Comment:
not sure how Qt handles via signals/slots but shouldn't this be a a `const HifiSockAddr&`